recipes/prometheus_client/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "prometheus_client" %}
+{% set version = "0.0.15" %}
+{% set build = 0 %}
+{% set sha256 = "823739260dab70a8f1d935e3cc9a1e5a55e22d8c9f33cdc6a0d81cdcb462a3dd" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/prometheus/client_python/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - twisted
+
+test:
+  imports:
+    - prometheus_client
+    - prometheus_client.bridge
+    - prometheus_client.twisted
+
+about:
+  home: https://github.com/prometheus/client_python
+  license: Apache Software License Version 2.0
```

Comment:
we should consider adding these to the linter so you don't have to go around commenting like this!